### ISSUE TITLE
LSICORR: correlation function not in outputted file

### ISSUE THREAD

**Issue by rerpha** [ISISComputingGroup/IBEX#7182]
As a developer I noticed that the LSI correlator had some issues when SANS2D tried to use it in action this cycle. 
Firstly, the files were being overwritten due to a typo in the code (fixed) - https://github.com/ISISComputingGroup/EPICS-LSICorrelator/pull/13
Secondly and most importantly, the correlation function is not being put in the output file. 

### Acceptance Criteria
_What is the acceptance criteria?_
- [ ] Above is fixed
- [ ] System test + unit tests written so it doesn't happen again

### Extra Information
From the support-issues channel on teams:
The correlation_data is completely blank. Otherwise it's putting things in the file correctly but the main job of the correlator really is to come up with this function so the data coming out of it is sort of rendered useless. 
    in other words https://github.com/ISISComputingGroup/EPICS-LSICorrelator/blob/master/data_file_interaction.py#L72 the correlation_data is `[] `  when printed to screen

## How to Test
_verbose instructions for reviewer to test changes
(**Add before making a PR**)_

**Comment by JackEAllen** [comment]
Seems to be working fine in emulator in that the correlation data is returning a populated array when running system tests and adding a print statement to view the array unlike the physical hardware. 
The LSI correlator should be physically available on the week commencing 6th June and also week commencing 20th June during shutdown period for testing.

**Comment by JackEAllen** [comment]
Booked some time to test with the hardware WC 20th June to try and resolve correlator function

**Comment by JackEAllen** [comment]
Hotfix exists on SANS2D for this fix: https://github.com/ISISComputingGroup/IBEX/wiki

**Comment by rerpha** [comment]
label automation is definitely not working  😢

**Comment by JackEAllen** [comment]
I've reached out to SANS2D to arrange some time to test the LSi correlator to figure out if the changes made to change from a minimum time lag to a maximum time lag are correct or whether the default minimum time lag set to 200 is not correct for SANS2D's use case.

**Comment by JackEAllen** [comment]
Booked time to test LSi Correlator for the 15th August

**Comment by JackEAllen** [comment]
> Booked time to test LSi Correlator for the 15th August

This has been rearranged again, waiting on SANS2D to provide us with a date to test

**Comment by JackEAllen** [comment]
There are several pre-set values that the sampling time (minimum time lag) can be set too: `12.5, 200, 400, 800, 1600, 3200`ns which IS should have the flexibility to select from. This means that the default value of 200 at the time of testing origionally in the summer may have been an inappropriate value to set too at the time of testing which is why the correlation array outputted may have been blank.
I will update the [WIKI page](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/LSi-Correlator#issuesgotchas) for the LSi Correlator with these value for future reference in the "Issues/Gotchas" section

**Comment by smk78** [comment]
Testing facilities can be provided on LOQ until mid-Oct. But it would be prudent to fully upgrade NDXLOQ & Ibex first.

**Comment by JackEAllen** [comment]
Requires Testing on LOQ as TS2 instruments are scheduled.  LOQ will required upgrading to Windows 10 and then IBEX V11 installed to match the environment it would be on if running on SANS2D. The upgrade to Windows 10 is in @ChrisM-S Win10 deployment schedule. An [issue](https://github.com/ISISComputingGroup/IBEX/issues/7365) has been created to deploy IBEX v11 to LOQ after this. Once updated, I will arrange a date for testing on LOQ
